generators/server/templates/src/main/java/package/config/_Constants.java

```diff
@@ -8,6 +8,7 @@
     //Regex for acceptable logins
     public static final String LOGIN_REGEX = "^[_'.@A-Za-z0-9-]*$";
     // Spring profile for development and production, see http://jhipster.github.io/profiles/
+    public static final String SPRING_PROFILE_TEST = "test";
```

Comment:
If that's just for the `CassandraTestConfiguration` I wouldn't bother with this: just put the `getPort` method in `CassandraConfiguration`... You have a check for `0` that can only be useful, there's no need to have a more complex configuration